packages/relay/src/lib/eth.ts

```diff
@@ -1722,7 +1722,7 @@ export class EthImpl implements Eth {
         const formattedTransactionId = formatTransactionIdWithoutQueryParams(submittedTransactionId);
         const contractResult = await this.mirrorNodeClient.repeatedRequest(
           this.mirrorNodeClient.getContractResult.name,
-          [formattedTransactionId, requestDetails],
+          [formattedTransactionId],
```

Comment:
Just checking is this the only place to remove it?
Other approach is to set IP undefined after it's utilized in the rate limiter if possible.

Reply:
No, its not the only place, i think there is one more, working on it
I will check your suggestion as well